etc/module.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
-    <module name="Rissc_Printformer" setup_version="100.8.69">
```

Comment:
please undo this change, the plan for now is:
as we said we merge tested things asap into master
then we retest on master
after test is successfully i would say better then we increase the setup-version number (in master)

so you can revert this file-change